docs/specifications/test-data/README.md

```diff
@@ -0,0 +1,53 @@
+# Test zones and test data for verification of test case implementation
+
+## Objective
+
+The purpose of the structure found here is to define test zones and test data to
+be able to test the correctness of the implementation of the Zonemaster
+[test cases]. The main use case is to be the basis for the [unit test] in the
+Zonemaster implementation. The second use case is to verify work in progress,
+e.g. implementation of new or updated test cases or updated test case
+implementation where the test case has not changed.
+
+There can be other use cases, but only these two use cases are covered here
+and in the test data specifications.
+
+
+## Test scenarios
+
+The goal of the test scenarios is to cover all reasonable contexts where
+different message tags are outputted when a test case is run on test zones. The
+message tags are defined in the test case specifications found via "[test cases]"
+and the scenarios are defined in the test case specific specifications for test
+data found via the [test-data] directory.
+
+In the test data specifications the scenarios are defined in two parts:
+
+* What messages from test case that are expected to be outputted and what
+  messages are expected not to be outputted when a test zone setup according to
+  the scenario is tested by the test case.
+* Specification of the zone setup for the scenario.
+
+
+## Test environment
+
+The tests against the test zones are assumed to be run in a closed environment
+with a private DNS tree to achieve full access to any zone. Configuration data
+and instructions to set this up are available elsewhere in this repository [to
+be completed and linked from here].
+
+All test zones are created under the non-existing .xa TLD except for a few cases
```

Comment:
Is there any explanation on the `.xa` TLD? Not that I'm against it (this is totally fine), just wondering if you considered using the reserved TLD `.test` ([RFC 2606 §2](https://www.rfc-editor.org/rfc/rfc2606#section-2))?

Reply:
.xa is shorter. Focus should be on the second and third levels. 

.xa is a "User-assigned: free for assignment at the disposal of users" and "User-assigned code elements are codes at the disposal of users who need to add further names of countries, territories, or other geographical entities to their in-house application of ISO 3166-1, and the ISO 3166/MA will never use these codes in the updating process of the standard."

https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Decoding_table